AdvancedDLSupport.Tests/Data/Interfaces/ISpanMarshallingTests.cs

```diff
@@ -27,6 +27,7 @@ namespace AdvancedDLSupport.Tests.Data
 {
     public interface ISpanMarshallingTests
     {
+        [NativeSymbol(CallingConvention = CallingConvention.StdCall)]
```

Comment:
Is this required?

Reply:
Yes, as the method is marked as __stdcall in C as well. And it will fail on x86 otherwise

Reply:
I'd prefer if we removed the __stdcall specifier in the C source instead, since that has its own implications and test cases. Is that doable?

Reply:
well yeah it's indeed doable :D